Code/GraphMol/ChemTransforms/ChemTransforms.cpp

```diff
@@ -32,7 +32,6 @@
 #include <RDGeneral/BadFileException.h>
 #include <GraphMol/ChemTransforms/ChemTransforms.h>
 namespace RDKit {
-namespace {
 void updateSubMolConfs(const ROMol &mol, RWMol &res,
```

Comment:
If we are exposing this to the public API, I think we need quite a bit more error checking in this code.

I can see a few paths where there might be unforeseen consequences and the name might be confusing as it's update SubMolConfs but takes removed atoms where I would expect it to take the submol atoms instead.

Reply:
> If we are exposing this to the public API, I think we need quite a bit more error checking in this code.
> 
> I can see a few paths where there might be unforeseen consequences and the name might be confusing as it's update SubMolConfs but takes removed atoms where I would expect it to take the submol atoms instead.

 I agree. I was trying to avoid code duplication but I think the inputs for this function are too obscure to be exposed. Do you think that we could make `updateSubConfs` a private function in ChemTransforms then make `RCore::extractCoreFromMolMatch` a friend (does that work across libraries?).  Alternatively, I can just copy the function over.

Reply:
How about putting it in the `details` namespace?
That's kind of a signal for "internal use" code.

Reply:
I've moved this function to details